### PR TITLE
Databases/week2

### PR DESCRIPTION
**Pull request by DarinkaGM**

### CODE REVIEW

File: databases/week2/HWexercise4.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  `topic` VARCHAR(255) NOT NULL,
+  `times_played` VARCHAR(255) NOT NULL,
+  PRIMARY KEY (`id`));
+  CREATE TABLE `Podcast` (
+  `id` INT NOT NULL AUTO_INCREMENT,
+  `title` VARCHAR(255) NOT NULL,
+  `suscribers` VARCHAR(255) NOT NULL,
+  `daily_listeners` VARCHAR(255) NOT NULL,
+  `genre_id` INT UNSIGNED NOT NULL,
+  PRIMARY KEY (`id`));
```
Comment (flemeur) [inline databases/week2/HWexercise4.sql:23]: There are quite a lot of things missing here.

1. The formatting is not great. Separate each table definition with newlines to make it easier to read.
2. You are missing the E/R diagram for the exercise. As mentioned in class it doesn't have to be anything fancy. A basic drawing is fine.
3. You haven't created any foreign keys, so what you have is essentially 4 tables with no connections to each other. I can see you have added a `genre_id` column in the `Podcast` table, but the actual foreign key definition is missing.
4. There is no many-to-many relation. You should have at least one.

Please don't be afraid to ask Marta or me, or your class mates, if you are unsure about something in the exercise.

**Review by flemeur — COMMENTED** [review]
Hi @DarinkaGM Here's my feedback. The first part of the homework went pretty well, but unfortunately the last part is missing a lot of things. Give it another try and don't hesitate to ask questions if you are unsure of something :)

File: databases/week2/HWexercise4.sql [hunk 2]
```diff
@@ -1,23 +1,43 @@
+SET NAMES utf8mb4;
+CREATE DATABASE Podcasts;
+USE Podcasts;
+
 CREATE TABLE `Host` (
   `id` INT NOT NULL AUTO_INCREMENT,
   `Name` VARCHAR(50) NOT NULL,
-  PRIMARY KEY (`id`));
+  `host_id` INT UNSIGNED,
```
Comment (martamatos) [inline databases/week2/HWexercise4.sql:8]: why a field named host_id in the host table?

File: databases/week2/HWexercise4.sql [hunk 3]
```diff
@@ -1,23 +1,43 @@
...
+  PRIMARY KEY (`id`))
+  ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+  
   CREATE TABLE `Genre` (
   `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,
   `Genre` VARCHAR(100) NOT NULL,
-  PRIMARY KEY (`id`));
-  CREATE TABLE `Episodes` (
+  `genre_id` INT UNSIGNED NOT NULL,
```
Comment (martamatos) [inline databases/week2/HWexercise4.sql:15]: same as above, why a field named genre_id in a table named genre?

File: databases/week2/HWexercise4.sql [hunk 4]
```diff
@@ -1,23 +1,43 @@
...
+  
   CREATE TABLE `Podcast` (
   `id` INT NOT NULL AUTO_INCREMENT,
   `title` VARCHAR(255) NOT NULL,
   `suscribers` VARCHAR(255) NOT NULL,
   `daily_listeners` VARCHAR(255) NOT NULL,
-  `genre_id` INT UNSIGNED NOT NULL,
-  PRIMARY KEY (`id`));
\ No newline at end of file
+  `podcast_id` INT UNSIGNED NOT NULL,
```
Comment (martamatos) [inline databases/week2/HWexercise4.sql:36]: podcast_id?

**Pull request merged**